packages/odyssey-react/src/components/Tag/Tag.theme.ts

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type { ThemeReducer } from "@okta/odyssey-react-theme";
+
+export const theme: ThemeReducer = (theme) => ({
+  // Color
+  TagBackgroundColor: theme.ColorPaletteNeutral100,
+
+  // Radius
+  TagBorderRadius: theme.BorderRadiusBase,
+  
+  // Space
+  ListSpaceMarginBlockEnd: theme.SpaceRemXs,
+  ListSpaceMarginInlineStart: theme.SpaceRemXs,
```

Comment:
consider removing all the `List` prefixes since this is targeting the root of the component itself. that `list` class should really be renamed to `root` in `Tag.module.scss` and `Tag.tsx` feel free to make that change!